### PR TITLE
[doc](udf) add some doc about java-udf handle static load

### PR DESCRIPTION
**Pull request by zhangstar333**

### CODE REVIEW

**Comment by luzhijing** [comment]
There are some conflicts and please resolve it.

**Review by morningman — CHANGES_REQUESTED** [review]
I will modify it later

**Review by luzhijing — APPROVED** [review]
LGTM

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query/udf/java-user-defined-function.md [hunk 1]
```diff
@@ -24,33 +24,74 @@ specific language governing permissions and limitations
...
+    "always_nullable"="true",
+    "type"="JAVA_UDF"
+);
+```
+
+### UDTF
 
-* 兼容性：使用 Java UDF 可以兼容不同的 Doris 版本，所以在进行 Doris 版本升级时，Java UDF 不需要进行额外的迁移操作。与此同时，Java UDF 同样遵循了和 Hive/Spark 等引擎同样的编程规范，使得用户可以直接将 Hive/Spark 的 UDF jar 包迁移至 Doris 使用。
+:::tip
+UDTF 自 Doris 2.1.3 版本开始支持
```
Comment (zhangstar333) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query/udf/java-user-defined-function.md:70]: PR not pick to 2.1.3 as have change the fe meta info.
so maybe release at a big version eg:4.0

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query/udf/java-user-defined-function.md [hunk 2]
```diff
@@ -24,33 +24,74 @@ specific language governing permissions and limitations
...
-Java UDF 为用户提供 UDF 编写的 Java 接口，以方便用户使用 Java 语言进行自定义函数的执行。相比于 Native 的 UDF 实现，Java UDF 有如下优势和限制：
 
-**1. 优势**
+Java UDF 为用户提供 UDF 编写的 Java 接口，以方便用户使用 Java 语言进行自定义函数的执行。
+
+Doris 支持使用 JAVA 编写 UDF、UDAF 和 UDTF。下文如无特殊说明，使用 UDF 统称所有用户自定义函数。
+
+## 创建 UDF
+
+实现的 jar 包可以放在本地也可以存放在远程服务端通过 HTTP 下载，但必须让每个 BE 节点都能获取到 jar 包。
```
Comment (zhangstar333) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query/udf/java-user-defined-function.md:39]: FE also need jar.
* 如果你是**本地路径**方式，这里数据库驱动依赖的 jar 包，**FE、BE 节点都要放置**

Comment (morningman) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query/udf/java-user-defined-function.md:39]: done

**Comment by zhangstar333** [comment]
this PR @morningman help to modify, and LGTM

**Pull request merged**